src/sources/libfaadloader.cpp

```diff
@@ -0,0 +1,190 @@
+    libnames << "/opt/local/lib/libfaad2.dylib";
+#else
+    DEBUG_ASSERT(!"OS not implementet");
+    return;
+#endif
+
+    for (const auto& libname : libnames) {
+        m_pLibrary.reset();
+        m_pLibrary = std::make_unique<QLibrary>(libname, 0);
+        if (m_pLibrary->load()) {
+            break;
+        }
+    }
+
+    if (!m_pLibrary->isLoaded()) {
+        kLogger.warning() << "Failed to load " << libnames << ", " << m_pLibrary->errorString();
+        m_pLibrary.reset();
+        return;
+    }
+
+    m_neAACDecOpen = reinterpret_cast<NeAACDecOpen_t>(
+            m_pLibrary->resolve("NeAACDecOpen"));
+    m_neAACDecGetCurrentConfiguration = reinterpret_cast<NeAACDecGetCurrentConfiguration_t>(
+            m_pLibrary->resolve("NeAACDecGetCurrentConfiguration"));
+    m_neAACDecSetConfiguration = reinterpret_cast<NeAACDecSetConfiguration_t>(
+            m_pLibrary->resolve("NeAACDecSetConfiguration"));
+    m_neAACDecInit2 = reinterpret_cast<NeAACDecInit2_t>(
+            m_pLibrary->resolve("NeAACDecInit2"));
+    m_neAACDecClose = reinterpret_cast<NeAACDecClose_t>(
+            m_pLibrary->resolve("NeAACDecClose"));
+    m_neAACDecPostSeekReset = reinterpret_cast<NeAACDecPostSeekReset_t>(
+            m_pLibrary->resolve("NeAACDecPostSeekReset"));
+    m_neAACDecDecode2 = reinterpret_cast<NeAACDecDecode2_t>(
+            m_pLibrary->resolve("NeAACDecDecode2"));
+    m_neAACDecGetErrorMessage = reinterpret_cast<NeAACDecGetErrorMessage_t>(
+            m_pLibrary->resolve("NeAACDecGetErrorMessage"));
+
+    if (    !m_neAACDecOpen ||
+            !m_neAACDecGetCurrentConfiguration ||
+            !m_neAACDecSetConfiguration ||
+            !m_neAACDecInit2 ||
+            !m_neAACDecClose ||
+            !m_neAACDecPostSeekReset ||
+            !m_neAACDecDecode2 ||
+            !m_neAACDecGetErrorMessage
+    ) {
+        kLogger.debug() << "NeAACDecOpen:" << m_neAACDecOpen;
+        kLogger.debug() << "NeAACDecGetCurrentConfiguration:" << m_neAACDecGetCurrentConfiguration;
+        kLogger.debug() << "NeAACDecSetConfiguration:" << m_neAACDecSetConfiguration;
+        kLogger.debug() << "NeAACDecInit2:" << m_neAACDecInit2;
+        kLogger.debug() << "NeAACDecClose:" << m_neAACDecClose;
+        kLogger.debug() << "NeAACDecPostSeekReset:" << m_neAACDecPostSeekReset;
+        kLogger.debug() << "NeAACDecDecode2:" << m_neAACDecDecode2;
+        kLogger.debug() << "NeAACDecGetErrorMessage:" << m_neAACDecGetErrorMessage;
+        m_neAACDecOpen = nullptr;
+        m_neAACDecGetCurrentConfiguration = nullptr;
+        m_neAACDecSetConfiguration = nullptr;
+        m_neAACDecInit2 = nullptr;
+        m_neAACDecClose = nullptr;
+        m_neAACDecPostSeekReset = nullptr;
+        m_neAACDecDecode2 = nullptr;
+        m_neAACDecGetErrorMessage = nullptr;
+        m_pLibrary->unload();
+        m_pLibrary.reset();
+        return;
+    }
+    kLogger.debug() << "Successfully loaded faad2 library " << m_pLibrary->fileName();
```

Comment:
- Increase log level to info
- Remove trailing space from message string -> "Successfully loaded library" (faad2 will appear as parts of the file name)